plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -260,6 +291,16 @@ struct flb_stackdriver *flb_stackdriver_conf_create(struct flb_output_instance *
         return NULL;
     }
 
+    /* resource labels */
+    flb_kv_init(&ctx->resource_labels_kvs);
```

Comment:
If the resource labels are processed after labels? Can kv pairs in labels be used in resource labels? 

I mean, for example, we have labels: a=b, and we also specify resource_labels as a=c, will it use the a=b kv pairs to form c=b here?

Reply:
They are completely independent, so the resource labels will have just `a=c` and not consider `a=b` at all.